specification/managementgroups/resource-manager/Microsoft.Management/preview/2024-01-01-preview/management.json

```diff
@@ -0,0 +1,2383 @@
+          {
+            "value": "noaccess"
+          },
+          {
+            "value": "view"
+          },
+          {
+            "value": "edit"
+          },
+          {
+            "value": "delete"
+          }
+        ]
+      },
+      "description": "The users specific permissions to this item."
+    },
+    "CheckNameAvailabilityRequest": {
+      "description": "Management group name availability check parameters.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "the name to check for availability"
+        },
+        "type": {
+          "type": "string",
+          "enum": [
+            "Microsoft.Management/managementGroups"
+          ],
+          "x-ms-enum": {
+            "name": "Type",
+            "modelAsString": false,
+            "values": [
+              {
+                "value": "Microsoft.Management/managementGroups"
+              }
+            ]
+          },
+          "description": "fully qualified resource type which includes provider namespace"
+        }
+      }
+    },
+    "PutApplication": {
+      "description": "The application details.",
+      "type": "object",
+      "x-ms-azure-resource": true,
+      "properties": {
+        "properties": {
+          "title": "properties",
+          "$ref": "#/definitions/CreateApplicationRequestProperties"
+        }
+      }
+    },
+    "CreateApplicationRequestProperties": {
+      "description": "Application creation request body parameters.",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "type": "string",
+          "description": "The provisioning state of the application. For example, Running",
+          "enum": [
+            "NotStarted",
+            "Running",
+            "Succeeded",
+            "Failed",
+            "Canceled"
+          ],
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The name of the application. For example, ApplicationTest1"
+        },
+        "memberOf": {
+          "title": "memberOf",
+          "$ref": "#/definitions/CreateApplicationMemberOfProperties"
+        }
+      }
+    },
+    "Application": {
+      "description": "The application details.",
+      "type": "object",
+      "x-ms-azure-resource": true,
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The unique resource identifier of the ARM resource application."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the ARM resource."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The type of Azure resource."
+        },
+        "properties": {
```

Comment:
Top-level properties (id, name, type, systemData, etc.) should come from v5 common types.